packages/astro/src/core/build/static-build.ts

```diff
@@ -31,6 +31,8 @@ export interface StaticBuildOptions {
 	viteConfig: ViteConfigWithSSR;
 }
 
+const MAX_CONCURRENT_RENDERS = 10;
```

Comment:
Any logic to `10` or just a good round number?

Reply:
I just tried a few and that one was the fastest.